src/SixLabors.Fonts/TextMeasurer.cs

```diff
@@ -58,6 +58,24 @@ public static FontRectangle MeasureBounds(ReadOnlySpan<char> text, TextOptions o
         /// <returns>Whether any of the characters had non-empty bounds.</returns>
         public static bool TryMeasureCharacterBounds(ReadOnlySpan<char> text, TextOptions options, out GlyphBounds[] characterBounds)
             => TextMeasurerInt.Default.TryMeasureCharacterBounds(text, options, out characterBounds);
+        
+        /// <summary>
+        /// Counts the lines from the generated layout.
+        /// </summary>
+        /// <param name="text">The text.</param>
+        /// <param name="options">The style.</param>
+        /// <returns>The count of lines used.</returns>
```

Comment:
```suggestion
```suggestion
        /// <summary>
        /// Gets the number of lines contained within the text.
        /// </summary>
        /// <param name="text">The text.</param>
        /// <param name="options">The style.</param>
        /// <returns>The line count.</returns>
```
```